index.js

```diff
@@ -1,5 +1,40 @@
 function scoreCalculator(arr) {
   // do work here
+  let points;
+  let pointsArr = [];
+
+  let bonus = arr.every(e => {
+    return e < 5;
+  });
+  if (bonus === true) {
+    arr.map(e => {
+      points = 10;
+      pointsArr.push(points);
+    });
```

Comment:
I don't think you need this array.map. I think it's throwing your point values off.